### PR TITLE
Assignment

### PR DESCRIPTION
**Pull request by hossein2024-hub**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
I have added some lines of code to make new directories, 
remove some data from a source directory to proper destinations,
copied data from one directory to another,
and create a new file contains all log files of users, events, and servers 

## What did you learn from the changes you have made?
I have learned how to mange files and directories in my computer by means of bash scripting, 
also, I got to manage my repository with the git version control programming

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
Not really

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
I was kind of confused when I was going to make my contributions in the assignment branch as I was not pretty clear about the mechanism of branching, pushing and pulling completely, I then read some articles and watched some videos and now I have a better understanding of the local repository, remote repository and how committing, pushing and pulling commands work to make connection between them 

## How were these changes tested?
Basically, I tried to run all the new lines through the VS code terminal, and finally, I also executed the bash file (assignment.sh) file itself. Throughout my testing process, I kept looking at the list of directories, files and the contents of the files/directories that was supposed to change with my lines of code. 

## A reference to a related issue in your repository (if applicable)
Not applicable

## Checklist
- [ Yes] I can confirm that my changes are working as intended

### CODE REVIEW

**Comment by monzchan** [comment]
Great job!

**Review by monzchan — APPROVED** [review]
